zio-http/jvm/src/main/scala/zio/http/netty/server/ServerChannelInitializer.scala

```diff
@@ -86,6 +87,16 @@ private[zio] final case class ServerChannelInitializer(
       case RequestStreaming.Enabled                        =>
       case RequestStreaming.Disabled(maximumContentLength) =>
         pipeline.addLast(Names.HttpObjectAggregator, new HttpObjectAggregator(maximumContentLength))
+      case RequestStreaming.Hybrid(maxAggregatedLength)    =>
+        pipeline.addLast(
+          Names.HttpObjectAggregator,
+          new HttpObjectAggregator(maxAggregatedLength) {
+            override def handleOversizedMessage(ctx: ChannelHandlerContext, oversized: HttpMessage): Unit = {
+              ctx.pipeline().replace(this, "chunkedWriter", new ChunkedWriteHandler())
```

Comment:
Won't this change the pipeline for all persisted connections to the server that are served through this channel?

Reply:
Ohh right, updating this